Edit wanikani.proto and generate Protobuf

UInt2048 opened the pull request:
Fix the issues in davidsansome/tsurukame#402 by re-generating from `wanikani.proto` and adding a setting.

d108 commented on ios/SettingsViewController.m:
```diff
@@ -205,6 +205,14 @@ - (void)rerender {
+  [model addItem:[[TKMSwitchModelItem alloc]
+                     initWithStyle:UITableViewCellStyleDefault
+                             title:@"Minimize review penalty"
+                          subtitle:@"Treat reviews answered incorrect multiple times as if "
```
This subtitle does not seem to show in the UI.

UInt2048 replied:
Fixed that - oops

d108 reviewed (changes requested):
@UInt2048 有難う御座います for this!

I noted in the review the subtitle is not appearing for me.

Would you also mind submitting your changes in a single commit so that I can do a "rebase and merge," and then it should appear on top of mine in the PR to https://github.com/davidsansome/tsurukame?

UInt2048 committed:
Generate Protobuf properly & make it a setting (#6)

* Support wrong answer counts greater than one

These revisions make changes to wrong answer handling for the following conditions.

- Incorrect answers beyond the first one for meanings and readings now get recorded and submitted to the Wanikani API.
- When a subject reappears, where previously a reading or meaning has been entered correctly, a prompt for the correct part will not appear.

When studying with grouped meanings and readings (back-to-back), entering an incorrect meaning, viewing its details, and then skipping the subject, it reappears at the end of the review queue. However, wrong answers for the subject not recorded resulted in SRS progress advancing mistakenly. It could reach the "burned" stage even after multiple incorrect guesses during the review.

* Support wrong answer counts greater than one

* We don't want to stop ask again later from working!

Co-authored-by: Daniel Zhang <d108@ikiapps.com>

UInt2048 committed:
Merge branch 'wrong_answer_handling' into 402-proto-fixes

UInt2048 committed:
Remove duplicate lines from bad merge

UInt2048 commented:
@d108 I tried, but GitHub didn't cooperate - it said I had to re-merge. If you squash and merge, it should do what you want.

d108 commented:
@UInt2048 Your efforts are appreciated. I hope the result is to your satisfaction concerning the appearance of your commit.
